From 3901bf424216bed790f25b52dda372817f4f1414 Mon Sep 17 00:00:00 2001
From: Violet Purcell <vimproved@inventati.org>
Date: Fri, 13 Dec 2024 12:41:21 -0500
Subject: [PATCH] replace usage of boost::asio::io_service with
 boost::asio::io_context

bost::asio::io_service was just a typedef to boost::asio::io_context and
was removed in 1.87.0.
--- a/src/input_common/udp/client.cpp
+++ b/src/input_common/udp/client.cpp
@@ -108,7 +108,7 @@ private:
     }
 
     SocketCallback callback;
-    boost::asio::io_service io_service;
+    boost::asio::io_context io_service;
     boost::asio::basic_waitable_timer<clock> timer;
     udp::socket socket;
 
-- 
2.47.1

